specification/metadata/metadata_schema_column_definition.md

```diff
@@ -0,0 +1,87 @@
+# FOCUS Metadata Schema Column Definition
+The FOCUS metadata schema column definition provides a list of the columns present in the FOCUS dataset along with metadata about the columsn. The intent of the schema metadata is to aid pracitioners to understand the structure of the FOCUS data, without the need to parse the included dataset. 
+
+
+## Logical Section: Schema 
+### Subsection: Schema Column Definition
+
+## Schema Definition Metadata
+
+ * ### column_name
+   * use: provides the date the schema was created
+   * type: STRING
+ * ### data_type
+   * use: data type of the column as provided and must be provided for all columns in the FOCUS dataset
+   * type: STRING
+ * ### max_length
+   * use: provides a maximum character length for columns whose data type where character length is relevant. This is typically provided for String type columns
+   * type: Integer
+* ### precision
+  * use: provide the maximum number of digits for the values in the column. This is typically provided for numeric columns
+  * type: Integer 
+* ### scale
```

Comment:
Maybe number_scale/NumberScale to be explicit

Reply:
Yeah I am good with that.